Fix different use of "remote" (#1270)
viamrobotics/docs: docs/program/sdks/_index.md

The easiest way to get started is to navigate to your robot's page on [the Viam app](https://app.viam.com/robots), select the **Code Sample** tab, select your preferred SDK, and copy the code generated for you.

> These boilerplate code samples import all of the necessary libraries and set up a client connection to your remote or local robot.

Reviewer: opt: do we need `remote or local` in the first place here? From the perspective of the Viam app, all robots are remote! Very optional, just a thought.

Author: ```suggestion
These boilerplate code samples import all of the necessary libraries and set up a client connection to your robot, whether your code runs on your robot or on a different machine.
```

> These boilerplate code samples import all of the necessary libraries and set up a client connection to your remote or local robot.
> These code snippets import all the necessary libraries and sets up a connection with the Viam app in the cloud.

Author: ```suggestion
These boilerplate code samples import all of the necessary libraries and set up a client connection to your robot through the Viam app in the cloud.
This is the easiest way to connect to your robot whether your code runs on your robot or on a different machine.
```

Author: or just 
```suggestion
These boilerplate code samples import all of the necessary libraries and set up a client connection to your robot through the Viam app in the cloud.
```
